ci/env/kava-internal-testnet/genesis.json

```diff
@@ -570,6 +570,10 @@
               "denom": "busd",
               "amount": "1000000000000000"
             },
+            {
+              "denom": "erc20/axelar/bnb",
```

Comment:
```suggestion
              "denom": "bnb",
```

see comment above

Reply:
Thanks @rhuairahrighairidh - will move above BUSD so the coins stay alphabetized